cmd/helpers.go

```diff
@@ -118,16 +134,22 @@ func InitializeCrossCompilation(verbose bool) error {
 }
 
 // BuildDocker builds the project using the cross compiling wailsapp/xgo:<xgoVersion> container
-func BuildDocker(binaryName string, buildMode string, projectOptions *ProjectOptions) error {
+func BuildDocker(binaryName string, buildDirectory string, buildMode string, projectOptions *ProjectOptions) error {
 	var packSpinner *spinner.Spinner
 	if buildMode == BuildModeBridge {
 		return fmt.Errorf("you cant serve the application in cross-compilation")
 	}
 
 	// Check build directory
-	buildDirectory := filepath.Join(fs.Cwd(), "build")
 	if !fs.DirExists(buildDirectory) {
-		fs.MkDir(buildDirectory)
+		_ = fs.MkDir(buildDirectory)
+	}
+
+	var outDir string
+	if filepath.IsAbs(buildDirectory) {
+		outDir = buildDirectory
+	} else {
+		outDir = filepath.Join(fs.Cwd(), buildDirectory)
```

Comment:
this means, if `buildDirectory` is default, or set by the user, but in both cases absolute, we do nothing, otherwise, we get the absoulte path of it, so that it can be passed to docker and resolved correctly.